ci-operator/config/mtc-qe/mtc-e2e-qev2/mtc-qe-mtc-e2e-qev2-master__mtc1.8-ocp4.14-lp-rosa-hypershift.yaml

```
@@ -0,0 +1,88 @@
+    name: openshift-cli-installer
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .
+  dockerfile_path: dockerfiles/interop/Dockerfile
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+    mtc-apps-deployer:
+      paths:
+      - destination_dir: .
+        source_path: /mtc-apps-deployer.tar.gz
+    mtc-python-client:
+      paths:
+      - destination_dir: .
+        source_path: /mtc-python-client.tar.gz
+  to: mtc-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: mtc-interop-aws-rosa
+  cron: 0 6 25 10 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      CLUSTER1_CONFIG: name-prefix=mtc-source;platform=hypershift;region=us-west-2;version=4.14;channel-group=nightly;compute-machine-type=m5.xlarge;timeout=2h;replicas=2
+      CLUSTER2_CONFIG: name-prefix=mtc-target;platform=hypershift;region=us-west-2;version=4.14;channel-group=nightly;compute-machine-type=m5.xlarge;timeout=2h;replicas=2
+      FIREWATCH_CONFIG: |
+        {
+        "failure_rules":
+        [
+          {"step": "cluster-install", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["!default"]},
+          {"step": "mtc-prepare-clusters", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster Configuration", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["!default"]},
+          {"step": "cucushift-*", "failure_type": "all", "classification": "Cluster Health Check", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
+          {"step": "cluster-uninstall", "failure_type": "all", "classification": "Infrastructure Deprovisioning - Cluster", "group": {"name": "cluster", "priority": 3}, "jira_additional_labels": ["!default"]},
+          {"step": "mtc-execute-tests", "failure_type": "pod_failure", "classification": "Test Execution", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["!default", "interop-tests"]},
+          {"step": "mtc-execute-tests", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "MIG", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["!default", "interop-tests"]}
+        ]
+        }
+      FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.14-lp","rosa-hypershift-lp","mtc-lp"]'
+      FIREWATCH_DEFAULT_JIRA_PROJECT: LPINTEROP
+      FIREWATCH_FAIL_WITH_TEST_FAILURES: "true"
+      MTC_VERSION: "1.8"
+      PULL_SECRET_NAME: pull-secret
+      S3_BUCKET_NAME: openshift-cli-installer-cspi-interop
+      S3_BUCKET_PATH: openshift-ci
+      TEST_PLATFORM: rosa
```

```suggestion
      TEST_PLATFORM: hypershift
```

Regarding the error you sent me in slack (`find: ‘tmp/clusters-data/rosa’: No such file or directory`). This is because the following lines are using the `TEST_PLATFORM` variable, and the openshift-cli-installer has set that to `hypershift`.

[Here is the code](https://github.com/openshift/release/blob/master/ci-operator/step-registry/mtc/prepare-clusters/mtc-prepare-clusters-commands.sh#L7-L9) that the error comes from:
```
tar -xzvf "${SHARED_DIR}/clusters_data.tar.gz" --one-top-leve=/tmp/clusters-data

SOURCE_CLUSTER_DIR=$(find tmp/clusters-data/${TEST_PLATFORM} -type d -name "${SOURCE_CLUSTER_PREFIX}*")
```

In the logs, the output shows the tar extraction, then it fails setting the `SOURCE_CLUSTER_DIR` variable. I know that `TEST_PLATFORM` should be `hypershift` because of the tar extraction logs:

```
Extracting cluster data
./
./hypershift/
```